example/components/connectorCards/WalletConnectCard.tsx

```diff
@@ -1,9 +1,12 @@
 import { URI_AVAILABLE } from '@web3-react/walletconnect'
 import { useEffect, useState } from 'react'
 
+import { MAINNET_CHAINS } from '../../chains'
 import { hooks, walletConnect } from '../../connectors/walletConnect'
 import { Card } from '../Card'
 
+const CHAIN_IDS = Object.keys(MAINNET_CHAINS).map(Number)
```

Comment:
This is repeated a lot, and if it were in a react component it would cause rerenders. Could you instead `export MAINNET_CHAIN_IDS = Object.keys(MAINNET_CHAINS).map(Number)` from `'../../chains'/?